vault.go

```diff
@@ -12,10 +14,31 @@ type plugin struct{}
 
 var KVSource plugin
 
-var vaultAddr = os.Getenv("VAULT_ADDR")
-var token = os.Getenv("VAULT_TOKEN")
+func getToken() (string, error) {
+	var t, exists = os.LookupEnv("VAULT_TOKEN")
```

Comment:
Huh, it really is `LookupEnv` but `Getenv`. Kind of a mixed bag of `*Env` and `*env`.

Reply:
Yup